convert2rhel/pkghandler.py

```diff
@@ -524,3 +526,19 @@ def install_additional_rhel_kernel_pkgs(additional_pkgs):
 def is_rhel_kernel_installed():
     installed_rhel_kernels = get_installed_pkgs_by_fingerprint(system_info.fingerprints_rhel, name="kernel")
     return len(installed_rhel_kernels) > 0
+
+
+def clear_yum_versionlock():
+    loggerinst = logging.getLogger(__name__)
+
+    if os.path.isfile(_VERSIONLOCK_FILE_PATH) and os.path.getsize(_VERSIONLOCK_FILE_PATH) > 0:
+        versionlock_file.backup()
```

Comment:
Move the backup below the `utils.ask_to_continue()`. It makes no sense to back the file up in case they don't want to continue.
As you can see from the output, the information about the backup just adds a noise around the important message under it:
```
[10/06/2020 14:04:37] TASK - [Prepare: Clear yum version locks] *********************************
Backing up /etc/yum/pluginconf.d/versionlock.list
Copying /etc/yum/pluginconf.d/versionlock.list to /var/lib/convert2rhel/
WARNING - yum versionlock plugin is in use. It may cause the conversion to fail.
Do you want to continue? This will clear package version locks.

Continue with the system conversion? [y/n]: y
```